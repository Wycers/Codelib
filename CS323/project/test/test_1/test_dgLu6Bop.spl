int test_5()
{
  int a = 009;
}