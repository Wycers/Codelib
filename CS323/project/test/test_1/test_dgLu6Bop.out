Error type A at Line 3: Wrong Form "009"